.changeset/thick-cats-compete.md

```diff
@@ -0,0 +1,14 @@
+---
+'astro': minor
+---
+
+The new rewrite APIs are deemed stable, which means that the experimental flag isn't needed anymore:
```

Comment:
Noting that I have some [standard guidance for writing the changeset when unflagging experimental features]() to get you started here!

Also, for convenience, the published URLs you might want to refer to here will be:

`Astro.rewrite` API reference: `https://docs.astro.build/en/reference/api-reference/#astrorewrite`
`## Rewrites` in the routing guide: `https://docs.astro.build/en/guides/routing/#rewrites`
`## Rewriting` in the middleware guide: `https://docs.astro.build/en/guides/middleware/#rewriting`